pkg/get/tools.go

```diff
@@ -3487,5 +3487,32 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
 				{{- end -}}
 				https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}-cli_{{.Version}}_{{$os}}_{{$arch}}.{{$extStr}}`,
 		})
+
+	tools = append(tools,
+		Tool{
+			Owner:       "replicatedhq",
+			Repo:        "replicated",
+			Name:        "replicated",
+			Description: "CLI for interacting with the Replicated Vendor API",
+			URLTemplate: `
+				{{$arch := "386"}}
+				{{ if (or (eq .Arch "x86_64") (eq .Arch "amd64")) -}}
+				{{$arch = "amd64"}}
+				{{- end -}}
+
+				{{$osStr := ""}}
+				{{- if eq .OS "darwin" -}}
+				{{$osStr = "darwin"}}
+				{{$arch = "all"}}
+				{{- else if eq .OS "linux" -}}
+				{{$osStr = "linux"}}
+				{{- end -}}
+
+				{{$extStr := "tar.gz"}}
+				{{$versionStr := .Version}}
+				{{$versionStr = ReplaceFirst $versionStr "v" "" 1}}
```

Comment:
Can you explain briefly for me why we needed ReplaceFirst for this CLI but haven't needed it anywhere else?

Reply:
Seems like

https://pkg.go.dev/strings#TrimPrefix

or

https://pkg.go.dev/strings#TrimLeft

Would have worked here too and been a bit more intuitive, if we're just taking off the "v"?

What did we do elsewhere? This pattern is not unique to this CLI.